app/javascript/src/component_activated_dialog.js

```diff
@@ -94,7 +94,7 @@ class Activator {
  * @classes (String) Classes added to button.
  **/
 function createActivator ($dialog, text, classes) {
-  const $activator = $("<button>\</button>")
+  const $activator = $("<button></button>")
```

Comment:
This looks like it was simply a typo rather that 'unrequired escaping', so nothing to worry about. 
Expect it didn't affect the program because we immediately overwrote the elements text with the next line.